src/dotnet/APIView/ClientSPA/src/app/_workers/apitree-builder.worker.ts

```diff
@@ -246,6 +250,9 @@ function addJustDiffBuffer() {
 function shouldAppendIfRowIsHiddenAPI(row: CodePanelRowData) {
   if (row.isHiddenAPI) {
     hasHiddenAPI = true;
+    if (isDiffRow(row)) {
+      hasHiddenAPIThatIsDiff = true;
```

Comment:
We should enable hidden APIs instead of asking user to enable the hidden APIs when we are sure that a hidden API is in diff.